### PR TITLE
Display areas

### PR DESCRIPTION
**Pull request by collinkallery**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling (Styling to new or existing files)

### Detailed Description
- Adding the areas to the application's state
- Fetch call for the areas endpoint, and a map over each area to retrieve the additional data from each specific area. 
- Passing the areas state down through the rendering of the areasContainer
- Map over each area within the areasContainer, add in props for each specific area, render each area. 
- Display the four areas in the areasContainer, add in styling for this container. 
- Add in styling for each area card. 

### Why is this change required? What problem does it solve?
- Allows a user to see the four neighborhoods in Denver they have to choose from. 
- Successfully fetches the data from the endpoint and displays it onto the DOM. 

### Was any previously written code changed?
- No 

### Directory modified:
- [x] - App
- [ ] - Login
- [x] - Area
- [x] - AreaContainer
- [ ] - Listing
- [ ] - ListingContainer

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -9,18 +9,47 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      current: 'login'
+      current: 'login',
+      areas: []
```
Comment (collinkallery) [inline src/components/App/App.js:13]: Add in state for the areas onto the application

File: src/components/App/App.js [hunk 2]
```diff
@@ -9,18 +9,47 @@ class App extends Component {
...
     }
   }
 
+  componentDidMount() {
```
Comment (collinkallery) [inline src/components/App/App.js:17]: Fetch call for the areas, maps over each area to retrieve additional data

File: src/components/App/App.js [hunk 3]
```diff
@@ -9,18 +9,47 @@ class App extends Component {
...
+          .then(completeAreaData => this.setState({areas: completeAreaData}))
+      })
+      .catch(err => console.error(err))
+  }
 
   render() {
     return (
       <div className="App">
-        <Login />
+        {/*<Login />*/}
```
Comment (collinkallery) [inline src/components/App/App.js:48]: Commented this out just so I could focus on styling for the areasContainer - tomorrow we will figure out how exactly to render the right page when.

File: src/components/Area/Area.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import  React from 'react';
+import './Area.css';
+
+const Area = (props) => {
```
Comment (collinkallery) [inline src/components/Area/Area.js:4]: Used props here as opposed to destructuring considering there were so many props - but I'm sure there's a better way to destructure this!

**Comment by C-Bandstra** [comment]
Thank you so much for the helpful comments. Ill be sure to add those to my future PR's

File: src/components/App/App.js (hunk 3 above)
Comment (C-Bandstra) [reply src/components/App/App.js:48]: I have the login functionality ready to go to display correct info on page change as well

**Pull request merged**